schemas/resources/schemas/grid/v0.0.1/metadata.json

```diff
@@ -11,7 +11,7 @@
     "GridDefinition": {
         "CoordinateReferenceSystemID": {
             "Type": "EPSG",
-            "Code": "EPSG:4326",
+			"Code": "EPSG:4326",
```

Comment:
Since this is under the `CoordinateReferenceSystemID` field shouldn't it be indented like the others?

Reply:
yes, it should.